test/instabug_test.dart

```diff
@@ -257,13 +259,48 @@ void main() {
   });
 
   test('[clearAllExperiments] should call host method', () async {
+    // ignore: deprecated_member_use_from_same_package
     await Instabug.clearAllExperiments();
 
     verify(
       mHost.clearAllExperiments(),
     ).called(1);
   });
 
+  test('[addFeatureFlags] should call host method', () async {
+    await Instabug.addFeatureFlags(
+      List.of([
+        FeatureFlag(name: 'name1', variant: 'variant1'),
+        FeatureFlag(name: 'name2', variant: 'variant2'),
+      ]),
+    );
+
+    verify(
+      mHost.addFeatureFlags(<String, String>{
+        "name1": "variant1",
+        "name2": "variant2",
+      }),
+    ).called(1);
+  });
+
+  test('[removeFeatureFlags] should call host method', () async {
+    const featureFlags = ["exp-1", "exp-2"];
+
+    await Instabug.removeFeatureFlags(featureFlags);
+
+    verify(
+      mHost.removeFeatureFlags(featureFlags),
+    ).called(1);
+  });
+
+  test('[clearAllFeatureFlags] should call host method', () async {
+    await Instabug.clearAllFeatureFlags();
+
+    verify(
```

Comment:
Can we review the featureFlags content to confirm the removal?

Reply:
There is no way to export feature tags from the native SDK to verify whether feature tags have been removed or added